widgets/src/lib/interfaces/Types.ts

```diff
@@ -57,8 +58,33 @@ export enum FinerGrainedTaskType {
 	"speech-enhancement"                                      = "Speech Enhancement",
 }
 
+export type PipelineModality = "audio" | "computer-vision" | "nlp" | "other";
```

Comment:
(nit) if we write "nlp", why do we write "computer-vision" and not "cv"?

Reply:
works for me !